OmniKitUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -459,10 +459,13 @@ class OmnipodSettingsViewController: RileyLinkSettingsViewController {
                 let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
                 
                 switch statusRow {
-                case .activatedAt:
+                case .activeTime:
                     let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-                    cell.textLabel?.text = LocalizedString("Active Time", comment: "The title of the cell showing the pod activated at time")
-                    cell.setDetailAge(podState.activatedAt?.timeIntervalSinceNow)
+                    cell.textLabel?.text = LocalizedString("Active Time", comment: "The title of the cell showing the pod active time")
+                    // Since the podState doesn't actually keep the pod's reported time active value returned in each response,
+                    // use a calculation based on a computed activated at time that is based on the dynamically updated expiresAt time
+                    // to have a consistent value and behavior when the pod's internal active time varies from the true elapsed time.
+                    cell.setDetailAge(podState.expiresAt?.addingTimeInterval(-Pod.nominalPodLife).timeIntervalSinceNow)
```

Comment:
Hmm, I think for this field, which is time since pairing, we should probably show the actual time from the phone, which is real, synchronized, time. I think showing expiration time using the skewed pod clock makes a lot of sense using the updated fields, as you do want to know when it's going to expire, and that will give you the best idea.  But time since pairing should probably not be skewed based on pod drift.

Reply:
The data reported in “Pod Settings” status section is displaying user friendly pod state (basically) as reported by the pod except for “Expires” which is a derived value that tracks the reported pod alive time and the “Active Time” which is derived even though it is actually reported by the pod (which is used to adjust the PodState.expiresAt value but not actually saved). Adjusting “Expires” using the pod’s reported active time makes it match the pod’s 72 hour expiration alert time, but the “Active Time” should also similarly match the pod’s values to prevent various anomalies with the “Pod Active” time not being an even hour value for the 72 hour expirationAdvsioryAlarm (in addition to being inconsistent with the “Expires” value), the 79 hour shutdownImminentAlarm, and the 80 hour end of service pod screamer. Additionally, if “Active Time” isn’t based the pod’s reported active time, it can differ from the Read Pod Status’s “Pod Active” value which is the pod’s reported active time (see https://github.com/ps2/rileylink_ios/issues/656 for an example). Another way to think about this is that a more natural implementation would have been to save the pod’s reported active time in PodState and just display this value (as a user might expect in this “Pod Settings” section), but this involves adding & maintaining another persistent PodState state value which I thought would be a harder sell since it wasn’t originally done this way instead of using a single one line change that has the same net result. Since all the other pod’s relative timing alarms & actions for the pod itself and in other places in Loop are all based on the pod’s notion of elapsed active time, there’s actually no good reason I can think of why the displayed “Active Time” should be based on the real, synchronized time instead of the pod’s reported active time. Additional note: this field isn’t actually the time since pairing, but rather the time since pod power up (i.e., insulin injection into the pod) which only the pod actually knows.